faraday/aprs.py

```diff
@@ -762,7 +786,8 @@ def main():
     """
 
     logger.info('Starting Faraday APRS-IS application')
-    sock = connectAPRSIS()
+    #sock = connectAPRSIS()
+    sock = ''
```

Comment:
Why pass `sock` to `aprs_worker` at all if it calls `connectAPRSIS()` itself?

Reply:
Good point. I used to need this but with the current implementation of the socket connections I do not. Removed!